pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.cdap.cdap</groupId>
   <artifactId>cdap</artifactId>
-  <version>6.8.0</version>
+  <version>6.8.1-SNAPSHOT</version>
```

Comment:
Is it the only place?

Reply:
Referring to other pom files: https://github.com/cdapio/cdap/pull/14546
Looks like there are bunch more to update

Reply:
you are right